modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -63,8 +65,9 @@ def get_pdf_data
           }
         end
 
-        def evss_service
-          ClaimsApi::EVSSService::Base.new(request)
+        # Creates a token OR gets existing one
+        def get_auth_token_service
+          @auth_token ||= BenefitsDocuments::Service.new.get_auth_token
```

Comment:
I wonder if we move this to app controller to be shared? we'll use the BD service elsewhere (claims_controller docs call for example)

Reply:
Yes that makes sense I will work on updating this for that.